### ISSUE TITLE
The markdownlint warning display is not working properly

### ISSUE THREAD

**Issue by roottool** [BoostIO/Boostnote#2996]
# Current behavior

![currently](https://user-images.githubusercontent.com/11808736/57403253-370caa00-7214-11e9-84d1-fbdf56fab40f.jpg)

The warning of red lines are warning for line 1 and the underline does not extend to the end of the line.

# Expected behavior

![expected](https://user-images.githubusercontent.com/11808736/57403575-fc574180-7214-11e9-859b-e4a6f9ab0417.jpg)

The warning displays at line 1 and the underline extends to the end of the line.

# Steps to reproduce

1. Make a new Markdown Note
2. Write the below code

```
# Tasks
- [ ] `Do something`
> - [ ] `Style for more lines`
> Some more lines  
```

3. Move the cursor to the warning icon⚠️ of line 2.

# Environment

- Version : Current Dev Build. It means current master source code.
- OS Version and name : Windows 10 Pro build 17763

# Finally

This bug has occurred by my fault.
I would like to fix this bug.
But It related to #2983 because the code of executing markdownlint is moved.
Therefore, I'm wondering whether to make a new PR or do it within this PR.

FYI: This bug has occurred by this code.
https://github.com/BoostIO/Boostnote/blob/377901606ef9c03bb694f503734ed9a95787a201/browser/components/CodeEditor.js#L61-L66

**Comment by ZeroX-DG** [comment]
You can just change it in the current PR. We'll also review it.

**Comment by Flexo013** [comment]
@roottool Was this resolved by https://github.com/BoostIO/Boostnote/pull/2983 ?

**Comment by roottool** [comment]
Sorry, I forgot to provide this information.
This bug is resolved by #2983.

## Step to reproduce

Writes the below code to the markdown note's first line in Boostnote.

```text
# Tasks
- [ ] `Do something`
```

## Current behavior (v0.11.16~)

The yellow underline extends to the end of the line.
The `# Tasks` line has a warning icon and a yellow underline.

## Past behavior (v0.11.15)

The yellow underline extends to the first character.
The `# Tasks` line doesn't have a warning icon and not to exist a yellow underline.

**Comment by Flexo013** [comment]
Thanks for the update!
